engine/dlib/src/remotery/defold.patch

```diff
@@ -86,10 +19,6 @@ index bb4d5f6ef4..3510cd3b55 100644
 +            name_len = strnlen_s(name, 256);
 +            property->nameHash = _rmt_HashString32(name, name_len, 0);
 +
-+            if (property->nameHash == 0 || name_len == 0) {
-+                dmLogError("REMOTERY: DEFOLD: RegisterProperty: The hash is 0x%08x! String is '%s' len: %u", property->nameHash, name, (uint32_t)name_len);
-+            }
-+
 +            QueueAddToStringTable(g_Remotery->mq_to_rmt_thread, property->nameHash, name, name_len, NULL);
 +/// END DEFOLD
```

Comment:
Removed the previous debug logging from remotely, since we haven't received any info from it.